### ISSUE TITLE
'no-descending-specificity' does not make sense with SASS

### ISSUE THREAD

**Issue by 66beta** [stylelint/stylelint#4885]
> Which rule, if any, is the bug related to?

no-descending-specificity

> What code is needed to reproduce the bug?

```sass
.header-button {
    & > button { 
        &:last-child { /*...*/ }    
    }
}
.search-bar {
    & > button { /*...*/ }
}
```

> What stylelint configuration is needed to reproduce the bug?

stylelint-config-standard

> Which version of stylelint are you using?

13.6.1

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

vscode with .stylelintrc.js
```js
module.exports = {
  extends: 'stylelint-config-standard',
};
```

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yes, it's related to SCSS nested properties.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

```
Expected selector ".search-bar > button" to come before selector ".header-button > button:last-child" (no-descending-specificity)
```

**Comment by jeddy3** [comment]
@66beta Thanks for the report and for using the template.

This is a [documented limitation](https://stylelint.io/user-guide/rules/no-descending-specificity#dom-limitations) of the rule. 

I suggest turning off the rule in your case.
